dir.props

```diff
@@ -22,9 +22,19 @@
 
   <!-- Tool versions -->
   <PropertyGroup>
-    <NuSpecReferenceGeneratorVersion>1.4.2</NuSpecReferenceGeneratorVersion>
+    <ProjectDir>$(MSBuildThisFileDirectory)</ProjectDir>
+    <SourceDir>$(ProjectDir)src\</SourceDir>
+    <PackagesDir>$(ProjectDir)packages\</PackagesDir>
+
+    <!-- Output directories -->
+    <BinDir>$(ProjectDir)bin\</BinDir>
+    <TestWorkingDir>$(BinDir)tests\</TestWorkingDir>
+    <BootstrapDestination>$(BinDir)Bootstrap\</BootstrapDestination>
+
+    <!-- Package version / directories -->
+    <BuildToolsVersion>1.0.25-prerelease-00199</BuildToolsVersion>
+    <CompilerToolsVersion>1.0.0</CompilerToolsVersion>
     <XunitVersion>2.1.0</XunitVersion>
-    <CompilerToolsVersion>1.2.0-beta1-20160108-01</CompilerToolsVersion>
```

Comment:
Updated it to 1.3.2 here and in the `project.json`, even though nothing seemed to care.